testnets/xiontestnet/assetlist.json

```diff
@@ -181,6 +174,52 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
       }
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/0166AE2224341A3F70943E315DAC6EDF012A638D0F9358794FF7525BA1DFC191",
+          "exponent": 0,
+          "aliases": [
+            "inj"
+          ]
+        },
+        {
+          "denom": "INJ",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/0166AE2224341A3F70943E315DAC6EDF012A638D0F9358794FF7525BA1DFC191",
+      "name": "Injective INJ token",
+      "display": "inj",
```

Comment:
display denom would be "INJ" all uppercase, not 'inj'

Reply:
make sense, thank you. updated to `INJ`